pages/api/landingPage.js

```diff
@@ -0,0 +1,59 @@
+import prisma from "client";
+
+export default async function handler(req, res) {
+  if (req.method === "POST") {
+    return await addContent(req, res);
+  } else if (req.method == "GET") {
+    return await readContent(req, res);
+  } else if (req.method == "PATCH") {
+    return await updateContent(req, res);
+  } else {
+    return res
+      .status(405)
+      .json({ message: "Method not allowed", success: false });
+  }
+}
```

Comment:
At some point, we will need to add a DELETE handler too

Reply:
Out of scope